init.lua

```diff
@@ -12,10 +14,23 @@ if enable_tnt == nil then
 	enable_tnt = minetest.is_singleplayer()
 end
 
+local sounds = {}
+local torch_item
```

Comment:
default and mcl are both optional. If neither is present, this causes the following unpleasant error: `Invalid crafting recipe (output="torch_bomb:torch_crossbow_wood")`

A possible solution to make it more end-user friendly: Add `assert(torch_item, "The mod 'torch_bomb' requires either 'default' or 'mcl' to function properly.")` further down in the file after `torch_item` is no longer modified.

Reply:
I think it was fixed with

`local torch_item = ""`

Reply:
From what I can understand from the code, that would allow players to ignite the node by punching it with bare hands. That's definitely not desired.